package.json

```diff
@@ -15,9 +15,20 @@
   "scripts": {
     "preinstall": "node bin/check-nvmrc.js",
     "prestart": "node bin/check-nvmrc.js",
-    "build": "node tasks/build.js",
+    "build": "node standard && tasks/build.js",
     "start": "node tasks/serve.js",
-    "lint": "gulp scss:lint"
+    "lint": "gulp scss:lint",
+    "test": "standard"
+  },
+  "standard": {
+    "globals": [
```

Comment:
This could be a source of bugs, since often GOV.UK needs to be reassigned to `window` etc.

Could we instead update the code so that with something like:

```js
const $ = window.jQuery
const GOV.UK = window.GOV.UK
```

Reply:
Agree. Did the same for vendor libs.